.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
I assumed this .gitignore would make future PRs and publishing a little easier as you wont have to be sure to clear these directories by hand.

Reply:
I always publish from github tarballs because of prior npm bugs, so I'm guaranteed to publish exactly what I expect.